packages/api/src/util/ensemble.js

```diff
@@ -94,7 +94,7 @@ async function registerOrganizationWebhook(farmId, organizationId, accessToken)
   const onResponse = async (response) => {
     await FarmExternalIntegrationsModel.updateWebhookAddress(
       farmId,
-      `${baseUrl}/sensors/add_reading/`,
+      `${baseUrl}/sensors/add_reading/:partner_id`,
```

Comment:
@AbdusShaikh I believe we need the actual partner id of 'Ensemble Scientific' here